src/lib/services/export-import-service.ts

```diff
@@ -140,10 +123,22 @@ export default class ExportImportService {
             ),
         );
         const result = {
-            features,
-            featureStrategies,
-            featureEnvironments,
-            contextFields: filteredContextFields,
+            features: features.map((item) => {
```

Comment:
remove irrelevant fields in the export that were also not compliant with openapi types Dates vs string